eox_tenant/test/test_signals.py

```diff
@@ -26,16 +26,6 @@ class StartTenantSignalTest(TestCase):
     Testing the signal sent at the start of every request
     """
 
-    def test_nothing_changes_studio(self):
```

Comment:
Can we add a test that makes sure that actually things in studio do change and thus perform as adverticed?

Reply:
This was discussed in a different channel and discarded